src/app/shared/entry-actions/workflow-actions.component.ts

```diff
@@ -65,4 +68,8 @@ export class WorkflowActionsComponent extends EntryActionsComponent implements O
     const prefix = this.sessionQuery.getValue().entryType === EntryType.Service ? ga4ghServiceIdPrefix : ga4ghWorkflowIdPrefix;
     this.refreshService.refreshWorkflow(prefix + this.workflow.full_workflow_path, versionName);
   }
+
+  delete() {
+    this.dialog.open(DeleteEntryDialogComponent, { width: '600px', data: this.workflow });
```

Comment:
Good thing Angular doesn't type check the `data` here with what's injected into the dialog, otherwise this would fail, as a `Workflow` is not an `Entry`, at least not until we solve dockstore/dockstore#5605. :)